src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    public void setSide(int side) {
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("This figure is Square with area "
+                + getArea() + " square units"
+                + " and properties: "
+                + getSide()
```

Comment:
```suggestion
                + side
```
you have direct access to the fields of this object